corehq/apps/builds/views.py

```diff
@@ -143,6 +144,8 @@ def import_build(request):
             return json_response({
                 'reason': 'build_number must be an int'
             }, status_code=400)
+    else:
+        build_number = random.randint(0, 100)
```

Comment:
This accounts for the optional build number in the UI. If none provided, populate it - the number itself does not matter.

Reply:
This would be a good code comment.

Reply:
This `else` block used to not exist, which suggests that it's actually valid to leave `build_number` blank, and indeed, looking at the model, I see that it's not a required field.  I know I said to use an arbitrary number, but I think blank is preferable if it doesn't cause any trouble.